guide/case-studies/blixt-wallet.md

```diff
@@ -0,0 +1,154 @@
+   retina = "/assets/images/guide/case-studies/blixt-wallet/blixt-ux-process@2x.jpg"
+   mobile = "/assets/images/guide/case-studies/blixt-wallet/blixt-ux-process-mobile.jpg"
+   mobileRetina = "/assets/images/guide/case-studies/blixt-wallet/blixt-ux-process.jpg"
+   alt-text = "Diagram of stages in the Blixt UX process"
+   width = 800
+   height = 929
+%}
+
+Since Blixt is an open source wallet we collected user research simply by going on their Discord channel and collecting comments and questions from users.
+
+## We discovered that Blixt users were struggling to open lightning channels.
+
+We created a lot of different user flows to find a solution for the best method to open a channel for a user new to lightning, and eventually settled on this one. We decided on this user flow as it educates users first and then takes them to setting up the channel itself. 
+
+
+## One of the biggest challenges we faced was…
+
+Explaining the concept of channel opening to people who are new to lightning. The concept of opening a channel to make a transaction is totally new to them. Next to this, some users might not be native English speakers and so using words and terms that communicate clearly to a larger group of people was essential.  
+
+## After re-designing the interface we tested the new design by doing usability tests. Here’s the process we followed:
+
+- **Plan:** We set out clear goals of what we wanted to understand and learn by conducting the tests. We also decided at this stage who we wanted to interview, as well as who we did not want to interview.
+- **User scenario:** We then went on to create a realistic user journey which the tester would walk through, with a clear goal that they would achieve in the end.
+- **Prototype:** The re-designed screenstates were then used to create a clickable prototype to test out the user journey we had in mind.
+- **Test script:** We then created a pre-screening questionnaire as well as a test script which we would use during the testing process.
+- **Testing:** The prototype was then tested on 4 people. During the usability test, they clicked through the prototype and we observed how they interacted with it.
+
+After watching the users testing the wallet we made some observations. Those observations were then grouped into different parts of the user flow. All of the data from the tests can be found [here](https://docs.google.com/document/d/1NuYAQX2c_AAnHq6VPMXrg3IxuzL6Pa517SY1Jx6wAs8/edit?usp=sharing). (We have not included the names of the testers to protect their privacy). 
+
+## Here’s what we learned…
+
+**TODO: Usuability Testing Data Overview graphic**
+
+- Users new to lightning are missing a crucial step, the why. They often lack understanding on why they need to open a channel before being presented with a screen to open one. Once this is clear it becomes easy for a user to understand that this step is necessary before transacting. 
```

Comment:
>  why they need to open a channel 

Would be good to link to a relevant section in the guide wrt channel openings, do we have something like this? I wasn't able to find it, cc @GBKS ?

Reply:
Generally, we try to hide anything related to channels in the daily spending wallet. The instance where it shows up is when you have to pay extra fees because a channel needs to be opened. There are visuals [here](https://bitcoin.design/guide/daily-spending-wallet/sending/#fees) and [here](https://bitcoin.design/guide/daily-spending-wallet/settings/#fees).

Those don't include explanations on what channels are. I think the easiest explainer is that a channel connects you to the network. Without a connection, you can't send or receive - easy to understand. The bigger your connection, the more sats can flow through it, etc.